class.json-api-endpoints.php

```diff
@@ -1592,7 +1592,7 @@ function get_closest_version_of_endpoint( $path, $request_method = 'GET' ) {
 				$endpoint_path = untrailingslashit( $endpoint['path'] );
 				$endpoint_path_regex = str_replace( array( '%s', '%d' ), array( '([^/?&]+)', '(\d+)' ), $endpoint_path );
 
-				if ( ! preg_match( "#^$endpoint_path_regex\$#", $path, $matches ) ) {
+				if ( ! preg_match( "#^$endpoint_path_regex\$#", $path, $string_matches ) ) {
```

Comment:
@zinigor do we even need this variable? Even if it's not provided, PHP won't overwrite the static `$matches`. Since it's never used again, it could be removed.